src/ArrayOfPrimeNumber.java

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by Yura on 02.05.2019.
+ */
+import java.util.Arrays;
+
+public class ArrayOfPrimeNumber {
+    public static void main(String[] args) {
+        int a = 100;
+        int[] myArray = new int[a];
+        int nextNumber = 2;
+        int counter = 0;
+        boolean isSimple;
+        while (counter < a) {
+            int i = 0;
+            isSimple = true;
+            while (isSimple && i <counter){
+                if (nextNumber % myArray[i] == 0) {
+                    isSimple = false;
+                }
+                i++;
+            }
+            if (isSimple) {
+                myArray[counter] = nextNumber;
+                counter++;
+                nextNumber++;
```

Comment:
if true
nextNumber++
if false
nextNumber++

so maybe it doesnt belong here? Extract